packages/dds/tree/src/core/tree/detachedFieldIndex.ts

```diff
@@ -137,15 +175,68 @@ export class DetachedFieldIndex {
 		return key;
 	}
 
+	/**
+	 * Returns all entries last created or used by the given revision.
+	 */
+	public getLatestRelevantRoots(revision: RevisionTag): [ForestRootId, Delta.DetachedNodeId][] {
```

Comment:
This seems like a confusing name for this function because it sounds as though the "latest relevant" part applies to the roots, which is not the case (the revision is the latest relevant to the roots). How about `getRootsLastTouchedByRevision`?